src/modes/direct_select.js

```diff
@@ -128,7 +131,10 @@ DirectSelect.onSetup = function(opts) {
     dragMoveLocation: opts.startPos || null,
     dragMoving: false,
     canDragMove: false,
-    selectedCoordPaths: opts.coordPath ? [opts.coordPath] : []
+    selectedCoordPaths: opts.coordPath ? [opts.coordPath] : [],
+    draggablePoint: opts.draggablePoint,
+    draggableLine: opts.draggableLine,
+    draggablePolygon: opts.draggablePolygon
```

Comment:
Shouldn't we provide a sensible default to these new options? As it stands, they'll default to `undefined`, which evaluates to `false`, which goes against the current default behaviour of the plugin where everything is draggable.